packages/workshop-app/app/routes/_app+/presence.ts

```diff
@@ -15,6 +15,25 @@ export function usePresencePreferences() {
 
 type User = Pick<ReturnType<typeof useUser>, 'id' | 'name' | 'avatarUrl'>
 
+export async function getPresentUsers(): Promise<User[]> {
+	try {
+		const presence = (await fetch(
+			// 'http://127.0.0.1:1999/party/epic-web-presence',
+			'https://epic-web-presence.kentcdodds.partykit.dev/party/epic-web-presence',
+			{
+				method: 'POST',
```

Comment:
We're not posting anything. Shouldn't this be a GET?

Reply:
yup, we could make this a GET 

I usually do a POST for live data to avoid caching snafus (from a painful past), but haha we control caching here too, so it can be a GET